src/hip/hip_build_utils.cpp

```diff
@@ -39,6 +39,48 @@
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_VERBOSE)
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_DUMP)
 
+#if MIOPEN_OFFLINE_COMPILER_PATHS_V2
+
+// List to hold Relative Directory Path
+// for each Compiler Flags from ROCM Install Path
+// The definition of this variable expected to be available in C/C++
+// code of the Target Library.
+static const char* MIOPEN_CMP_FLAG_REL_PATHS[MIOPEN_CMP_MAX_Flag] = {
+    "llvm/bin/clang", "bin/clang-ocl", "llvm/bin/clang++", "llvm/bin/clang-offload-bundler"};
+
+// Global variable List to hold the Compiler Flag Paths.
+char MIOPEN_CMP_FLAGS[MIOPEN_CMP_MAX_Flag][PATH_MAX_LEN] = {0};
```

Comment:
See my previous comment.

Reply:
These lines of code changed in latest version